php/libraries/LorisForm.class.inc

```diff
@@ -1235,11 +1235,14 @@ class LorisForm
 
                 // Check if date fields have errors and add them to error list
                 if ($el['type'] === 'date') {
-                    $date        = (
+                    $date = (
                         $values[$elName] ?
                             $values[$elName] :
                         $this->getValue($el['name'])
                     );
+                    if (empty($date)) {
+                        break;
```

Comment:
won't this break out of the foreach loop and cause the rest of the elements to not be validated?

I think this should be `continue`, but maybe I'm misunderstanding this code.